payloads/v8/application.ts

```diff
@@ -2,7 +2,8 @@
  * Types extracted from https://discord.com/developers/docs/resources/application
  */
 
-import type { Snowflake } from '../../globals';
+import type { OAuth2Scopes } from './index';
```

Comment:
can you import it from its actual file instead, or consolidate all other imports from this one?